### PR TITLE
[Hugo] Solution page PoC

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: layouts/shortcodes/product-grid.html [hunk 1]
```diff
@@ -1,24 +1,6 @@
-{{- $dataArray := slice }}
+{{- $dataArray := partial "folder-data.html" }}
 {{- $availableOptions := slice -}}
 
-{{- range readDir "./data/" -}}
```
Comment (kodster28) [inline layouts/shortcodes/product-grid.html:4]: Moved to a partial :) so we can reuse across shortcodes

File: config/_default/config.toml [hunk 2]
```diff
@@ -7,7 +7,7 @@ disableHugoGeneratorInject = true
 enableRobotsTXT = false
 summaryLength = 30
 disableAliases = true
-disableKinds = ["taxonomy", "term"]
```
Comment (kodster28) [inline config/_default/config.toml:10]: We need to surface `term` pages, but then adding a blank .html template so they don't get rendered :)

File: layouts/partials/folder-data.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{{- $dataArray := slice }}
```
Comment (kodster28) [inline layouts/partials/folder-data.html:1]: This is where the logic from `product-grid` shortcode went

File: data/use-cases.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
...49L45.8509 14.0989L48.6263 16.8743L54.6197 10.8809L51.8443 8.10549Z" fill="black"/><path d="M62.113 28.799H53.637V32.725H62.113V28.799Z" fill="black"/><path d="M10.363 28.971H1.88696V32.897H10.363V28.971Z" fill="black"/><path d="M12.0339 8.2275L9.25836 11.0028L15.2523 16.9972L18.0278 14.2219L12.0339 8.2275Z" fill="black"/><path d="M32 12.22C27.9634 12.2191 24.0359 13.5295 20.8085 15.9539C17.5811 18.3783 15.2286 21.7854 14.1051 25.6624C12.9816 29.5394 13.1479 33.6765 14.5789 37.4508C16.01 41.2252 18.6284 44.4325 22.04 46.59V53.98L24 55.95H40L41.96 53.98V46.59C45.3715 44.4325 47.9899 41.2252 49.421 37.4508C50.8521 33.6765 51.0184 29.5394 49.8948 25.6624C48.7713 21.7854 46.4188 18.3783 43.1914 15.9539C39.9641 13.5295 36.0365 12.2191 32 12.22ZM39.06 43.75L38.04 44.31V52.02H25.96V44.31L24.94 43.75C22.0737 42.1819 19.811 39.7043 18.5088 36.7079C17.2067 33.7114 16.9391 30.3668 17.7483 27.2014C18.5576 24.0361 20.3976 21.2302 22.9781 19.2264C25.5586 17.2226 28.7328 16.1349 32 16.1349C35.2671 16.1349 38.4413 17.2226 41.0218 19.2264C43.6023 21.2302 45.4423 24.0361 46.2516 27.2014C47.0608 30.3668 46.7933 33.7114 45.4911 36.7079C44.1889 39.7043 41.9262 42.1819 39.06 43.75Z" fill="black"/></svg>
```
Comment (KianNH) [inline data/use-cases.yml]:
```suggestion
logo: <svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" fill="none"><path fill="#000" d="M40.022 60.021H23.983v3.926h16.039v-3.926ZM33.877 0h-3.926v8.477h3.926V0ZM51.844 8.105 45.851 14.1l2.775 2.775 5.994-5.993-2.776-2.776ZM62.113 28.799h-8.476v3.926h8.476v-3.926ZM10.363 28.971H1.887v3.926h8.476v-3.926ZM12.034 8.227l-2.776 2.776 5.994 5.994 2.776-2.775-5.994-5.995Z"/><path fill="#000" d="M32 12.22a18.626 18.626 0 0 0-9.96 34.37v7.39L24 55.95h16l1.96-1.97v-7.39A18.627 18.627 0 0 0 32 12.22Zm7.06 31.53-1.02.56v7.71H25.96v-7.71l-1.02-.56a14.71 14.71 0 1 1 14.12 0Z"/></svg>
```

Ran the SVG through https://jakearchibald.github.io/svgomg/

**Review by KianNH — APPROVED** [review]

**Pull request merged**